<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interfaz</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QStackedWidget" name="wiewsSW">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="loginPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QFrame" name="loginFrame">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <property name="formAlignment">
            <set>Qt::AlignCenter</set>
           </property>
           <item row="0" column="0" colspan="2">
            <widget class="QGroupBox" name="userGP">
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <italic>true</italic>
               <bold>true</bold>
               <underline>false</underline>
              </font>
             </property>
             <property name="title">
              <string>Login</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="loginEmailLabel">
                <property name="font">
                 <font>
                  <family>Bahnschrift Light</family>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                  <underline>false</underline>
                 </font>
                </property>
                <property name="text">
                 <string>Email:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="loginEmailLE">
                <property name="font">
                 <font>
                  <family>Bahnschrift Light</family>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                  <underline>false</underline>
                 </font>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="passwordLabel">
                <property name="font">
                 <font>
                  <family>Bahnschrift Light</family>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                  <underline>false</underline>
                 </font>
                </property>
                <property name="text">
                 <string>Password: </string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="passwordLE">
                <property name="font">
                 <font>
                  <family>Bahnschrift Light</family>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                  <underline>false</underline>
                 </font>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="loginPB">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="font">
                 <font>
                  <family>Bahnschrift Light</family>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                  <underline>false</underline>
                 </font>
                </property>
                <property name="text">
                 <string>Login</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QGroupBox" name="newUserGB">
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Bahnschrift Light</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <italic>true</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="title">
              <string>New here?</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="newUserLabel">
                <property name="font">
                 <font>
                  <family>Bahnschrift Light</family>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>User name: </string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="newUserLE">
                <property name="font">
                 <font>
                  <family>Bahnschrift Light</family>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="emailLabel">
                <property name="font">
                 <font>
                  <family>Bahnschrift Light</family>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Email: </string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="emailLE">
                <property name="font">
                 <font>
                  <family>Bahnschrift Light</family>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="newPasswordLabel">
                <property name="font">
                 <font>
                  <family>Bahnschrift Light</family>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Password: </string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="newPasswordLE">
                <property name="font">
                 <font>
                  <family>Bahnschrift Light</family>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QPushButton" name="createPB">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="font">
                 <font>
                  <family>Bahnschrift Light</family>
                  <pointsize>12</pointsize>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Create</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="nilePage">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="0">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Todos los departamentos</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Alimentos y Bebidas</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Libros</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Electrónicos</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hogar Y Cocina</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Deporte y Aire Libre</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="filtrarLable">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Filtrar:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QPushButton" name="srchPB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="maxPricePB">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Mayor Precio</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLineEdit" name="srchLE"/>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="minPricePB">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Menor Precio</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="srchLable">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Buscar:</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0" colspan="3">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1162</width>
             <height>184</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QGridLayout" name="productsGL"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="15" column="0" colspan="3">
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1162</width>
             <height>183</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QGridLayout" name="recommendGL"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>50</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>true</strikeout>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>NILE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
